libqtile/widget/battery.py

```diff
@@ -190,6 +190,12 @@ class _LinuxBattery(_Battery, configurable.Configurable):
             None,
             "Name of file with the current power draw in /sys/class/power_supply/battery_name",
         ),
+        (
+            "full_not_charging",
+            False,
+            "Some batteries report their status as 'Not charging' when full, rather than 'Full'. "
+            "If the widget reports an unknown state when full, try changing this to 'True'.",
```

Comment:
> If the widget reports an unknown state when full, try changing this to 'True'.

Can we do this for them instead of providing a new option?

Reply:
Well, we *could*... My concern is if there's a scenario that the battery reports "Not charging" when it's not actually full. I agree, it would be nice not to have this as an option.

The spec says this:
```
What:		/sys/class/power_supply/<supply_name>/status
Date:		May 2007
Contact:	linux-pm@vger.kernel.org
Description:
		Represents the charging status of the battery. Normally this
		is read-only reporting although for some supplies this can be
		used to enable/disable charging to the battery.

		Access: Read, Write

		Valid values:
			      "Unknown", "Charging", "Discharging",
			      "Not charging", "Full"
```
So "Not charging" should be separate to a "Full" state. I wonder if the "Full" state is returned if the `capacity` is less than a certain level so it's not actually full even though it will no longer accept a charge.

Let me go back to the OP and get them to check the capacity when the charge stops.